src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -20,28 +20,29 @@ def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_w
     provider = 'Microsoft.DBForMySQL' if isinstance(client, ServersOperations) else 'Microsoft.DBforPostgreSQL'
     parameters = None
     if provider == 'Microsoft.DBForMySQL':
-        import azure.mgmt.rdbms.mysql.models
-        parameters = azure.mgmt.rdbms.mysql.models.ServerForCreate()
-        parameters.sku = azure.mgmt.rdbms.mysql.models.Sku()
-        parameters.properties = azure.mgmt.rdbms.mysql.models.ServerPropertiesForDefaultCreate()
-        parameters.properties.storage_profile = azure.mgmt.rdbms.mysql.models.StorageProfile()
+        from azure.mgmt.rdbms import mysql
+        parameters = mysql.models.ServerForCreate(sku=mysql.models.Sku(name=sku_name),
```

Comment:
Recommend more compact syntax:
```Python
from mysql.models import ServerForCreate, Sku, ServerPropertiesForDefaultCreate, StorageProfile
parameters = ServerForCreate(
  sku=Sku(name=sku_name), 
  properties=ServerPropertiesForDefaultCreate(
    administrator_login=administrator_login,
    administrator_login_password=administrator_login_password, 
    version=version, 
    ssl_enforcement=ssl_enforcement, 
    storage_profile=StorageProfile(
      backup_retention_days=backup_retention, 
      geo_redundant_backup=geo_redundant_backup, 
      storage_mb=storage_size)
    ),
    location=location, 
    tags=tags
) 
```

Reply:
The pylint throws errors when I do this.